try:
    print(x)

except:
    print("An error occurred")

finally:
    print("success")


try:
    num1 = 20
    num2 = 2
    print(num1 / num2)

except:
    print("An error occurred")